pkg/services/live/live.go

```diff
@@ -286,7 +265,5 @@ func handleLog(msg centrifuge.LogEntry) {
 		loggerCF.Error(msg.Message, arr...)
 	case centrifuge.LogLevelInfo:
 		loggerCF.Info(msg.Message, arr...)
-	case centrifuge.LogLevelNone:
```

Comment:
What's the rationale behind removing handling of this level?

Reply:
Centrifuge never calls log handler func when LogLevelNone set since it means no logging, though `golang-ci` critisized this change since the new code does not cover all possible cases - so I added default branch in latest commit.

Reply:
Yeah, it's best to make sure all switch branches are handled, to avoid brittle code.